apps/buildog/src/components/sidebar.tsx

```diff
@@ -1,41 +1,52 @@
+"use client";
+
 import { cn } from "@repo/ui/lib/utils";
 import { Button } from "@ui/components/button";
 import { usePathname, useRouter } from "next/navigation";
 
-export function Sidebar({ className }: { className?: string }) {
+interface SidebarProps {
+  className?: string;
+  organizationId: string; // Accept organizationName as a prop
+}
+
+export function Sidebar({ className, organizationId }: SidebarProps) {
   const pathname = usePathname();
   const router = useRouter();
+
   const routes = {
     Main: {
       children: [
         {
           name: "Blog",
           icon: "",
-          route: "/blog/",
+          route: `/organizations/${organizationId}/blog`, // Dynamic route
```

Comment:
You don't have to say this is a Dynamic route with comment _for all_
```suggestion
          route: `/organizations/${organizationId}/blog`,
```